cds_ils/migrator/items/api.py

```diff
@@ -75,10 +75,18 @@ def set_document_pid(record):
 
 def import_items_from_json(dump_file, rectype="item"):
     """Load items from json file."""
-    dump_file = dump_file[0]
     model, provider = model_provider_by_rectype(rectype)
-
     with click.progressbar(json.load(dump_file)) as bar:
+        error_logger.error(
```

Comment:
minor: Should it be info instead of error?
```suggestion
        error_logger.info(
```
question: Could you quickly explain when should we use each logger?

Reply:
I use error because somewhere we have hardcoded logger level lower than info and logs with .info do not appear. I will test `LOGGING_CONSOLE_LEVEL = 'INFO'` and if it works I will change these accordingly